users/urls.py

```diff
@@ -1,9 +1,8 @@
+from django.urls import include
 from rest_framework.urlpatterns import format_suffix_patterns
 
 from django.conf.urls import url
-from django.conf.urls import include
 from django.urls import path
-
```

Comment:
You should keep a line of break between Django specific imports and custom application imports.